devices/MS5611/MS5611.nuspec

```diff
@@ -0,0 +1,42 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>nanoFramework.Iot.Device.Ms5611</id>
+    <version>$version$</version>
+    <title>nanoFramework.Iot.Device.Ms5611</title>
+    <authors>nanoFramework project contributors</authors>
+    <owners>nanoFramework,dotnetfoundation</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <license type="file">LICENSE.md</license>
+    <releaseNotes>
+    </releaseNotes>
+    <readme>docs\README.md</readme>
+    <developmentDependency>false</developmentDependency>
+    <projectUrl>https://github.com/nanoframework/nanoFramework.IoT.Device</projectUrl>
+    <icon>images\nf-logo.png</icon>
+    <repository type="git" url="https://github.com/nanoframework/nanoFramework.IoT.Device" commit="$commit$" />
+    <copyright>Copyright (c) .NET Foundation and Contributors</copyright>
+    <description>This package includes the .NET IoT Core binding Iot.Device.Ms5611 for .NET nanoFramework C# projects.</description>
+    <summary>Iot.Device.MS5611 assembly for .NET nanoFramework C# projects</summary>
```

Comment:
@josesimoes I think the summary one is not used anymore, correct?

Reply:
That is correct. Summary tag has been deprecated.

Reply:
Should I remove the summary node, or make it empty, or leave it as it is?

Reply:
@rkalwak please just remove the tag entirely. You'll probably think "but that's on the other nugets" and you're right. The thing is that this has been deprecated recently and we are fixing that as we touch the other nuspec files.

Reply:
ok, done